src/Core/Models/StaticStore/Plans/TeamsStarterPlan.cs

```diff
@@ -28,6 +28,8 @@ public TeamsStarterPlan()
 
         PasswordManager = new TeamsStarterPasswordManagerFeatures();
         SecretsManager = new TeamsStarterSecretsManagerFeatures();
+
+        LegacyYear = 2024;
```

Comment:
This hides the plan on the frontend [here](https://github.com/bitwarden/clients/blob/main/apps/web/src/app/billing/organizations/organization-plans.component.ts#L775)